bin/platformio-custom.py

```
@@ -244,26 +271,30 @@ def as_list(val):
 
 prefsLoc = projenv["PROJECT_DIR"] + "/version.properties"
 verObj = readProps(prefsLoc)
-print(f"Using meshtastic platformio-custom.py, firmware version {verObj['long']} on {env.get('PIOENV')}")
+print(
+    f"Using meshtastic platformio-custom.py, firmware version {verObj['long']} on {env.get('PIOENV')}"
+)
 
 # get repository owner if git is installed
 try:
     r_owner = (
         subprocess.check_output(["git", "config", "--get", "remote.origin.url"])
```

Ruff (0.15.20)

[error] Starting a process with a partial executable path

(S607)